wolfssl/ssl.h

```diff
@@ -553,9 +553,11 @@ struct WOLFSSL_X509_STORE {
 #ifdef HAVE_EX_DATA
     WOLFSSL_CRYPTO_EX_DATA ex_data;
 #endif
-#if (defined(OPENSSL_EXTRA) || defined(WOLFSSL_WPAS_SMALL)) && defined(HAVE_CRL)
+#ifdef HAVE_CRL
     WOLFSSL_X509_CRL *crl; /* points to cm->crl */
 #endif
+    wolfSSL_Mutex   refMutex;   /* reference count mutex */
```

Comment:
Limit the inclusion of these fields.
Specifically SINGLE_THREADED.

Reply:
Limited all `refMutex` fields I could find with `SINGLE_THREADED`.